### PR TITLE
Create the-journal-on-efficiency-and-responsibility-in-education-and-…

### PR DESCRIPTION
**Pull request by SuperDomek**
Uploaded a new style for the ERIES journal.

### CODE REVIEW

File: the-journal-on-efficiency-and-responsibility-in-education-and-science.csl [hunk 1]
```diff
@@ -0,0 +1,636 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>The Journal on Efficiency and Responsibility in Education and Science</title>
+    <title-short>ERIES</title-short>
+    <id>http://www.zotero.org/styles/the-journal-on-efficiency-and-responsibility-in-education-and-science</id>
+    <link href="http://www.zotero.org/styles/the-journal-on-efficiency-and-responsibility-in-education-and-science" rel="self"/>
+    <author>
```
Comment (POBrien333) [inline the-journal-on-efficiency-and-responsibility-in-education-and-science.csl]: Can you also include a link to the template style you used?

**Comment by POBrien333** [comment]
the style is not validating because of the et-al settings. :)

**Comment by SuperDomek** [comment]
Oh, interesting, though. I validated it through the validator and created in the editor. :/ It doesn't look that hard to fix from the log. The original style is APA, I'll put it there. Thanks for your time.

**Comment by adam3smith** [comment]
I fixed the et-al -- we have some tests on the settings sanity that go beyond pure validation and you had an `et-al-min` without an `et-al-use-first`,  which we don't allow.

**Comment by rmzelle** [comment]
Thanks! Sorry for the wait!

**Pull request merged**